devel/python/python/ert_gui/models/connectors/run/iterated_ensemble_smoother.py

```diff
@@ -52,8 +52,9 @@ def createTargetCaseFileSystem(self, phase):
     def analyzeStep(self, target_fs):
         self.setPhaseName("Analyzing...", indeterminate=True)
         source_fs = self.ert().getEnkfFsManager().getCurrentFileSystem()
-        success = self.ert().getEnkfSimulationRunner().smootherUpdate(source_fs , target_fs)
-
+        
+        es_update = ESUpdate( self.ert() )
```

Comment:
Maybe use the same pattern as `getEnkFsManager()` and `getEnkfSimulationRunner()`